ui/components/MesheryPatterns.js

```diff
@@ -34,7 +34,7 @@ import AddIcon from '@mui/icons-material/AddCircleOutline';
 import React, { useEffect, useRef, useState } from 'react';
 import { UnControlled as CodeMirror } from 'react-codemirror2';
 import Moment from 'react-moment';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
```

Comment:
@yash37158 is this intentionally included?

Reply:
@yash37158 i guess this came from your previous commit. Revert them please